### PR TITLE
Jonathan McKinley Updated Photo 

### PR DESCRIPTION
**Pull request by Pandarati**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [X] I have read the instructions on the README file before submitting my application. 
- [X] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [X] I have used the Markdown file template to add my information to the Year Book.
- [X] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [X] I understand I should not tag or add a reviewer to this Pull Request.
- [X] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [X] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/Pandarati/Pandarati.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Jonathan McKinley
+institution: Florida A&M University 
+profile_pic: mckinley_headshot.jpg
+quote: Life isn’t how many breaths you take, but it’s the moments that take your breath away. # no longer than 100 characters
```
Comment (Mohammed-Shoaib) [inline _data/Pandarati/Pandarati.md:5]: Change the ’ to single quote ' as it is a non-ascii character.

File: _data/Pandarati/Pandarati.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Jonathan McKinley
+institution: Florida A&M University 
+profile_pic: mckinley_headshot.jpg
```
Comment (Mohammed-Shoaib) [inline _data/Pandarati/Pandarati.md:4]: Incorrect file name, please do one of the following:
1. Change `mckinley_headshot.jpg` to `mckinley_headshot_1.JPG` in the markdown file.
2. Rename `mckinley_headshot_1.JPG` to `mckinley_headshot.jpg`.
I'd suggest you rename the file!

Comment (juanpflores) [inline _data/Pandarati/Pandarati.md:4]:
```suggestion
profile_pic: mckinley_headshot_1.JPG
```

File: _data/Pandarati/Pandarati.md (hunk 1 above)
Comment (juanpflores) [inline _data/Pandarati/Pandarati.md:5]:
```suggestion
quote: Life isn’t how many breaths you take, but it’s the moments that take your breath away.
```

**Pull request merged**